include/mesh.tcc

```diff
@@ -1885,9 +1885,92 @@ void mpm::Mesh<Tdim>::create_nodal_properties() {
   }
 }
 
+// Create the nodal properties' map for discontinuity
+template <unsigned Tdim>
+void mpm::Mesh<Tdim>::create_nodal_properties_discontinuity() {
+  // Initialise the shared pointer to nodal properties
+  if (nodal_properties_ == nullptr)
+    nodal_properties_ = std::make_shared<mpm::NodalProperties>();
+
+  // Check if nodes_ is empty and throw runtime error if they are
+  if (nodes_.size() != 0) {
+    // Compute number of rows in nodal properties for vector entities
+    const unsigned nrows = nodes_.size() * Tdim;
+    // Create pool data for each property in the nodal properties struct
+    // object. Properties must be named in the plural form
+    nodal_properties_->create_property("mass_enrich", nodes_.size(), 1);
```

Comment:
I would change this to "masses_enrich" to keep all as plural

Reply:
How about keep the same name with the quantities in` node.h`? Always use the singular form. which one will be better?